modules/introduction/pages/whats-new.adoc

```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
 The JavaScript UDF libraries can now be stored within collection scopes, allowing access to be restricted to users with the necessary security permissions.
 
 UI Support::
-Javascript libraries and functions can now be added through the Query tool in the adminstration console.
+Javascript libraries and functions can now be added through the Query tool in the administration console.
 
 === Improvements to Full-Text Search
 
+A number of improvements have been made to Full-Text Search:
+
 Croatian Language Support::
 The FTS engine now supports the Croatian language for search-term input.
 
 Improved Search in the UI::
 The FTS UI will now accept complex full text search terms as well as basic search strings.
 
+File-Based Rebalance in FTS::
+Prior to version{nbsp}7.1, FTS index partitions were rebuilt from scratch on each node during rebalance operations. 
+Under version{nbsp}7.1, the index partition files are copied to the nodes, allowing for faster rebalancing.
+
+Increase in the Number of Full-Text Indexes Supported::
+FTS now supports up 750 indexes per node.
+
 === Index Service Enhancements
 
 The Index Service now provides:
 
-* _Smart Batching_, which allows the rebuilding of indexes, during rebalance, to be maintained at an optimal performance-level.
+Smart Batching:: 
```

Comment:
These belong to GSI .. and don't think it has to come under FTS changes?

Reply:
Ya, not applicable to FTS.

Reply:
Don't worry about those; they're not in your section. I just reformatted them while I was there.

Sorry for the confusion.